pkg/karmadactl/taint_test.go

```diff
@@ -0,0 +1,585 @@
+package karmadactl
+
+import (
+	"reflect"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+
+	clusterv1alpha1 "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
+)
+
+func TestValidate(t *testing.T) {
+	cases := []struct {
+		taintOpt    CommandTaintOption
+		description string
+		expectFatal bool
+	}{
```

Comment:
For the following two errs, I think some cases can be added:

https://github.com/karmada-io/karmada/blob/ec60c2d103843e4c729b536f873303590c59a861/pkg/karmadactl/taint.go#L146

https://github.com/karmada-io/karmada/blob/ec60c2d103843e4c729b536f873303590c59a861/pkg/karmadactl/taint.go#L163

Reply:
thanks for your reviews, @XiShanYongYe-Chang :)

Ok, I will do it.